Build/NuGet/.pack-version

```diff
@@ -1 +1 @@
-1.10.1
+1.10.2-rc
```

Comment:
Can you please rebase this commit to remove the `-rc` and update the commit message?